kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelService.kt

```diff
@@ -199,15 +195,6 @@ class TunnelService : VpnService() {
         return START_STICKY
     }
 
-    // Happens when a user removes the VPN configuration in the System settings
```

Comment:
Was this comment not true?

Reply:
Hmm let me test this. The docs aren't clear on when this callback is invoked. It wasn't occurring in the typical tunnel lifecycle

Reply:
Yeah, it doesn't seem to ever be called both when disconnecting from the App UI or by forgetting the VPN in system settings.